PyNite/Member3D.py

```diff
@@ -1186,6 +1186,92 @@ def PlotDeflection(self, Direction):
         Member3D.__plt.xlabel('Location')
         Member3D.__plt.title('Member ' + self.Name)
         Member3D.__plt.show()
+    
+#%%   
+    def RelativeDeflection(self, Direction, x):
+        
+        d = self.d()
+        d1y = d[1,0]
+        d2y = d[7,0]
+        d1z = d[2,0]
+        d2z = d[8,0]
+        L = self.L()
+
+        """
+        Returns the relative deflection at a point along the member's length
+        
+        Parameters
+        ----------
+        Direction : string
+            The direction in which to find the relative deflection. Must be one of the following:
+                "dy" = Deflection in the local y-axis
+                "dz" = Deflection in the local x-axis
+        x : number
+            The location at which to find the relative deflection
+        """
```

Comment:
The docstring should be directly below the method definition to function properly.